README.md

```diff
@@ -30,6 +22,28 @@ sudo apt install -y wget
 sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
 wget https://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
 ```
+
+## Binary install
+
+OpenRMF binary packages are available for Ubuntu Focal 20.04 for the `Foxy`, `Galactic` and `Rolling` releases of ROS 2. Most OpenRMF packages have the prefix `rmf` on their name, therefore, you can find them by them by searching for the pattern `ros-<ro2distro>-rmf`, e.g., for galatic it would be:
+
+```bash
+apt-cache search ros-galactic-rmf
+```
+
+A good way to install the `rmf` set of packages in one go is to install the `rmf_demos` package. This will pull all the rest of the OpenRMF packages as a dependency. As an example, for Galactic, you would run:
+
+```bash
+sudo apt install ros-galactic-rmf-demos
```

Comment:
Do you wanna mention `ros-galactic-rmf-demos-gz` or `ros-galactic-rmf-demos-ign`?

Reply:
misspell in the readme: `rmf_dmeos_gz`. Otherwise, the rest looks good. :+1: